app/src/pages/SingleContent.vue

```diff
@@ -15,6 +15,7 @@ import RelatedContent from "../components/content/RelatedContent.vue";
 import VerticalTagViewer from "@/components/tags/VerticalTagViewer.vue";
 import Link from "@tiptap/extension-link";
 import LImage from "@/components/images/LImage.vue";
+import { useHead } from "@vueuse/head";
```

Comment:
According to the plugin's GitHub page, it has been replaced with Unhead/vue, so the best will be to use the new package.

But to keep our application as lightweight as possible, I would suggest not using a plugin to set the title and metadata, but rather set the title as currently done in line 44, and setting the metadata as per https://stackoverflow.com/questions/2568760/is-it-possible-to-use-javascript-to-change-the-meta-tags-of-the-page